gc.c

```diff
@@ -13566,8 +13572,10 @@ str_len_no_raise(VALUE str)
     return (int)len;
 }
 
-const char *
-rb_raw_obj_info(char *buff, const int buff_size, VALUE obj)
+#define OBJ_INFO_EXT 1 << 0
```

Comment:
I think it is better to be parenthesized.

```suggestion
#define OBJ_INFO_EXT (1 << 0)
```

Or,

```suggestion
static const int OBJ_INFO_EXT = 1 << 0;
```

Reply:
oh yeah, macros definitely need to be parenthesized, my bad. I'll change it to use the `static const int` version, because macros are the wrong kind of fun :)